src/hotspot/share/oops/klass.hpp

```diff
@@ -569,6 +572,7 @@ class Klass : public Metadata {
       return true;
     }
   }
+#endif
```

Comment:
Can you add an // INCLUDE_CDS here?  If they're more than 10 lines apart, uncommented endifs can be really confusing.  This one is sort of borderline far away.  Thanks.

Reply:
Fixed.